pkgs/tools/misc/rtz/default.nix

```diff
@@ -41,9 +47,16 @@ rustPlatform.buildRustPackage rec {
 
   buildFeatures = [ "web" ];
 
+  # ${swagger-ui} is read-only and the copy made by the build script
+  # is as well. Remove it so that checks can copy it again.
+  checkPhase = ''
+    find target -name $(basename ${swagger-ui}) -delete
```

Comment:
This would usually go in `preCheck`, so you don't have to repeat the default on the next line.

Reply:
Indeed. Done.